check_project.sh

```diff
@@ -10,15 +10,64 @@
 # EMBArk is licensed under MIT
 #
 # Author(s): Michael Messner, Pascal Eckmann
+# Contributer(s): Benedikt Kuehne
 
-# Description:  Check all shell and python scripts
-
+# Description:  Check all scripts and templates(Django gets its own unit-tests)
 
+RED='\033[0;31m'
 GREEN='\033[0;32m'
 ORANGE='\033[0;33m'
 BOLD='\033[1m'
 NC='\033[0m' # no color
 
+# checks js-scripts with jshint for errors
+# config @ ./embark/static/.jshintrc
+jscheck(){
```

Comment:
could we include a quick check if jshint is already available. If it is not available give a note that the installer script should be used